### PR TITLE
Adds basic shortcut handlers

### PR DESCRIPTION
**Pull request by alschmiedt**
Adds the navigation controller class. This is what the user will use to add the plugin: 

```
const navigationController = new NavigationController();
navigationController.init();
navigationController.addWorkspace(workspace)
```

The init method will also add all necessary shortcuts, however that will be added in a later PR. This PR is just focused on the methods that add shortcut handlers.

### CODE REVIEW

File: plugins/keyboard-navigation/src/navigation_controller.js [hunk 1]
```diff
@@ -0,0 +1,208 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
```
Comment (rachel-fenichel) [inline plugins/keyboard-navigation/src/navigation_controller.js]: 2021 :)

File: plugins/keyboard-navigation/src/navigation_controller.js [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Registers all of the keyboard shortcuts that are necessary for
+ * navigating blockly using the keyboard.
+ * @author aschmiedt@google.com (Abby Schmiedt)
+ */
+
+import '../src/gesture_monkey_patch';
```
Comment (rachel-fenichel) [inline plugins/keyboard-navigation/src/navigation_controller.js]: Why not just `./gesture_monkey_patch`?

File: plugins/keyboard-navigation/src/navigation_controller.js [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+  disable(workspace) {
+    this.navigation.disableKeyboardAccessibility(workspace);
+  }
+
+
+  /**
+   * Removes all the keyboard navigation shortcuts.
+   * @public
+   */
+  dispose() {
```
Comment (rachel-fenichel) [inline plugins/keyboard-navigation/src/navigation_controller.js:220]: This removes the shortcuts but leaves the handlers in place, right?

**Review by rachel-fenichel — APPROVED** [review]

File: plugins/keyboard-navigation/src/navigation_controller.js (hunk 3 above)
Comment (alschmiedt) [reply plugins/keyboard-navigation/src/navigation_controller.js:220]: Yup, good point. Added a `removeShortcutHanlders` method.

File: plugins/keyboard-navigation/src/navigation_controller.js (hunk 2 above)
Comment (alschmiedt) [reply plugins/keyboard-navigation/src/navigation_controller.js]: Fixed.

File: plugins/keyboard-navigation/src/navigation_controller.js (hunk 1 above)
Comment (alschmiedt) [reply plugins/keyboard-navigation/src/navigation_controller.js]: fixed.

**Pull request merged**